addon/utils/render-tooltip.js

```diff
@@ -26,13 +26,34 @@ export default function renderTooltip(domElement = {}, options = {}) {
     options.event = 'hover';
   }
 
+  if (options.duration && typeof options.duration === 'string') {
+    options.duration = parseInt(options.duration, 10);
+    if (isNaN(options.duration) || !isFinite(options.duration)) {
```

Comment:
Sorry to be a stickler but can you add block space around the if blocks? (i.e. each `if` should havean extra line above it)